kafka-commons/src/main/java/uk/gov/homeoffice/digital/sas/kafka/serializer/CustomJsonSerializer.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.homeoffice.digital.sas.kafka.serializer;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.springframework.kafka.support.serializer.JsonSerializer;
+
+public class CustomJsonSerializer<T> extends JsonSerializer<T> {
+
+  public static final ObjectMapper mapper = createMapper();
+
+  public static final ObjectMapper createMapper() {
+    ObjectMapper mapper = new ObjectMapper();
+    mapper.registerModule(new JavaTimeModule());
+    mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
+    return mapper;
+
+  }
```

Comment:
After long attempts with @JWeston-HO, although we managed to inject the object mapper as a Java bean in a custom serialiser bean. The latter wasn't picked up for Kafka serialisation. The reason being that when serialisers/deserialisers are set using properties, Kafka instantiates them, not Spring (serialiser is specified using this property `spring.kafka.producer.value-serializer`). See this answer for further details: https://stackoverflow.com/a/52077820/2219361

So if we were to specify the serialiser as a Spring bean, we would have to revert back to using Producer Factory, which we have moved on from since this PR: https://github.com/UKHomeOffice/callisto-timecard-restapi/pull/83